lib/controller.js

```diff
@@ -641,6 +663,213 @@ controller.tesselFirmwareVerion = function(opts) {
         });
     });
 };
+controller.menu = {
```

Comment:
Let's remove all `menu`-related code and put it in it's own PR. It's not specifically related to SSH sessions.